src/App.elm

```diff
@@ -34,7 +31,7 @@ init : () -> ( Model, Cmd Msg )
 init _ =
     let
         model =
-            { openDefinitions = []
+            { openDefinitions = OpenDefinitions.empty
```

Comment:
Later on `init` will be called with a list of hashes from the url and this would become `OpenDefinitions.loading hashes`